### DISCUSSION TITLE
Interested in Extending Interoperability and On The Fly Transformations

### DISCUSSION THREAD

**Discussion by imperorrp** [MDAnalysis/mdanalysis#4485] [GSoC Discussions]
Hi people! I'm Ratish, an undergrad from BIT Mesra, India. Most of my prior development experience includes some freelance web3/crypto work (making bots and scripts, working with various 3rd party APIs and SDKs) and some web dev. 

I haven't worked on much scientific software development (yet), but I've been working with LAMMPS, Atomsk, Ovito and other molecular dynamics adjacent tools recently as part of some materials science research (in High Entropy Alloys, so solid state MD and not for example biochemistry MD) I'm working on, so I do have some idea of this area. 

I'm interested in the 2nd and 3rd projects as of now. I'll look into them further and read up more on the MDAnalysis documentation. Looking forward to sending a pre-proposal and making a contribution soon hopefully

**Comment by hmacdope** [comment]
Awesome! Looking forward tor reading your pre-proposal.
